OpenKeychain/src/main/java/org/sufficientlysecure/keychain/securitytoken/SecurityTokenHelper.java

```diff
@@ -206,7 +206,7 @@ public void connectToDevice(final Context ctx) throws IOException {
         mPw1ValidatedForDecrypt = false;
         mPw3Validated = false;
 
-        if (mOpenPgpCapabilities.isHasAESSM()) {
+        if (mOpenPgpCapabilities.isHasSCP11bSM()) {
```

Comment:
Before this change, establishement of SCP11b was tried some with tokens that do not support which generated a stack trace and an error message in the log (example [here](https://github.com/open-keychain/open-keychain/issues/2049)), but without any other inconvenient as the execution continued normally. With this change, establishement of SCP11b secure messaging is triggered only with tokens that support it, which is now possible thanks to an update of the OpenPGP card specification (v3.3).